app/views/RoomActionsView/ActionsList/RoomItem.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';

```

Comment:
Bad component naming.
`RoomItem` is one of most used components in the project.
Every `Cmd+P` on VSCode would show this one on the list.